### ISSUE TITLE
[Bug]: EKS composition aggregated connection secret not populated

### ISSUE THREAD

**Issue by MichaelVL** [aws-samples/crossplane-aws-blueprints#34]
### Welcome to AWS Blueprints for Crossplane!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/aws-samples/crossplane-aws-blueprints/issues) and didn't find any.

### AWS Blueprints for Crossplane Release version

SHA c5cfbe0

### What is your environment, configuration and the example used?

The EKS composition in `compositions/aws-provider/eks` used with the example from `examples/aws-provider/composite-resources/eks/eks-claim.yaml`, which has the following connection secret config:

```
  writeConnectionSecretToRef:
    name: xplane-eks-cluster
```

### What did you do and What did you see instead?

After deployment, the secret `xplane-eks-cluster` is empty.

### Additional Information

```shell
The problem is due to the `connectionDetails` on the cluster resource in `compositions/aws-provider/eks/eks-managed-node-group.yaml` which have the following definition for `kubeconfig`:


      connectionDetails:
        - name: value
          fromConnectionSecretKey: kubeconfig
```

The key `value` is not aligned with the XRD which is why the secret becomes empty:

```
  connectionSecretKeys:
    - kubeconfig
```
```

**Comment by haarchri** [comment]
@nabuskey think this issue is fixed with #35 ?

**Comment by nabuskey** [comment]
Yep that is correct. I guess it wasn't linked correctly. Thanks for the heads up!
